tests/unit/test_create.py

```diff
@@ -68,13 +68,13 @@ class Config:
 
     str_value: Grammar[str]
     str_value_or_none: Grammar[str] | None = None
-    optional_str_value: Optional[Grammar[str]] = None
+    optional_str_value: Grammar[str] | None = None
```

Comment:
I'm surprised `tox run -m format` didn't fix these in 57b50fab

Reply:
They're marked as unsafe fixes in ruff, so you'd have to run ruff with `--unsafe-fixes`.

This can be a great thing to do in its own commit though when making big sweeping changes, but you need to review each change.

Reply:
ruff didn't touch these because this is actually inside of a string (it's hard to tell on the diff)
but thanks for calling attention to this, these are ones that I wanted to preserve as `Optional` etc on purpose